.gitignore

```diff
@@ -1,3 +1,5 @@
 /target/
 **/*.rs.bk
 Cargo.lock
+# vim swap files
+*.swp
```

Comment:
Please add this in a separate commit, or even as a separate PR against master :)